notebooks/8-rotation-speeds-pt2/interm_statistics.ipynb

```diff
@@ -0,0 +1,2490 @@
+{
```

Comment:
<p>should remove error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>